plugins/modules/clustering/pacemaker/pacemaker_cluster.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Mathieu Bultel <mbultel@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: pacemaker_cluster
+short_description: Manage pacemaker clusters
+author:
+- Mathieu Bultel (@mbultel)
+- Hani Audah (@haudah)
+description:
+   - This module can manage a pacemaker cluster and nodes from Ansible using
+     the pacemaker cli.
+options:
+    state:
+      description:
+        - Indicate desired state of the cluster
+      choices: [ cleanup, offline, online, restart, present, absent ]
+      type: str
+    node:
+      description:
+        - Specify which node of the cluster you want to manage. None == the
+          cluster status itself, 'all' == check the status of all nodes.
+      type: str
+    pcs_user:
+      description:
+        - The username used to authenticate with new nodes being added to the cluster
+      type: str
+    pcs_password:
+      description:
+        - The password used to authenticate with new nodes being added to the cluster
+      type: str
+    timeout:
+      description:
+        - Timeout when the module should considered that the action has failed
+      default: 300
+      type: int
+    force:
+      description:
+        - Force the change of the cluster state
+      type: bool
+      default: 'yes'
```

Comment:
```suggestion
      default: true
```
Always use `true`/`false` for YAML booleans.